build-configs/BuildConfigType.js

```diff
@@ -3,11 +3,21 @@
 import type { ThemeType } from './ThemeType'
 import type { TranslationsType } from 'translations'
 
+// Prevent enabled intro slide in combination with a preselected city.
+// If you change this make sure you are not navigating to the landing screen upon closing the intro slides.
+export type SelectedCityType = {|
+  introSlides: false,
+  selectedCity: string | null // If set, the city is automatically preselected. Changing it is not possible for users.
+|} | {|
+  introSlides: true,
+  selectedCity: null
```

Comment:
maybe `undefined` is better here. Thay way you can just omit the prop

Reply:
Actually flow is stupid and gives you this anyway:
Error:(25, 62) Cannot assign object literal to `commonIntegreatE2eBuildConfig` because property `fixedCity` is missing in  object literal [1] but exists in  `FeatureFlagsType` [2] in property `featureFlags`.